content/tutorials/reactor-event-zapier.textile

```diff
@@ -19,15 +19,17 @@ reading_time: 30
 languages:
   - cplusplus
 tags:
-    - Reactor Intergration
+    - Reactor Integration
     - Webhooks
     - Zapier
 ably_product: reactor
 ---
 
-The Ably Reactor provides a way to trigger events and to stream data from Ably's pub/sub channels. In this tutorial, we'll see how to build an IoT based realtime attendance system that sends a message to a Slack channel upon detecting an access card. We'll use "Ably's MQTT adapter":/mqtt to communicate with the IoT device and Zapier to send messages to Slack.
+In this tutorial, we'll see how to build an IoT based realtime attendance system that sends a message to a Slack channel upon detecting an access card. 
 
-h2(#step1-prerequisites). Step 1 - Prerequisites
+We'll use "Ably's MQTT adapter":/mqtt to communicate with the IoT device and Zapier to send messages to Slack. Ably's "Reactor Integrations"https://www.ably.io/reactor provide a way to trigger events and to stream data from Ably's pub/sub channels. 
```

Comment:
`"Reactor Integrations"https://www.ably.io/reactor` is missing seperating `:` between text/link.